contractcourt/chain_watcher.go

```diff
@@ -308,7 +308,7 @@ func (c *chainWatcher) closeObserver(spendNtfn *chainntnfs.SpendEvent) {
 			return
 		}
 
-		// We'll not retrieve the latest sate of the revocation store
+		// We'll now retrieve the latest sate of the revocation store
```

Comment:
you can also fix the `s/sate/state` typo while you're at it 😄

Reply:
Lol. There's already too many 't's in "Latest sate"  :P